File: easy/Rightmost-Char/Rightmost Char.cs
using System.IO;
using System.Collections.Generic;

class Program
{
    static void Main(string[] args)
    {
        using (StreamReader reader = File.OpenText(args[0]))
        while (!reader.EndOfStream)
        {
            string line = reader.ReadLine();
            if (null == line)
                continue;
            // do something with line
            ShowCharacter(line);
        }
    }
    
    public static void ShowCharacter(string line){
            line = line.Trim();
            int leng=line.Length;
            if(leng>1){
                int pos = -1;
                char ch = line[leng-1];
                for(int i=0;i<leng-2;i++)if(line[i]==ch)pos = i;
                System.Console.WriteLine(pos);
            }
        }
    
}

File: easy/Rightmost-Char/Rightmost Char.java
import java.io.*;
public class Main {
    public static void main (String[] args) throws IOException {
        File file = new File(args[0]);
        BufferedReader buffer = new BufferedReader(new FileReader(file));
        String line;
        while ((line = buffer.readLine()) != null) {
            line = line.trim();
            // Process line of input Here
            ShowCharacter(line);
        }
    }
    
    public static void ShowCharacter(String line){
            line = line.trim();
            int leng=line.length();
            if(leng>1){
                int pos = -1;
                char ch = line.charAt(leng-1);
                for(int i=0;i<leng-2;i++)if(line.charAt(i)==ch)pos = i;
                System.out.println(pos);
            }
        }
}
